encoding/hex_test.ts

```diff
@@ -21,14 +21,14 @@ const testCases = [
 
 const errCases: [string, ErrorConstructor, string][] = [
   // encoded(hex) / error / msg
-  ["0", RangeError, ""],
-  ["zd4aa", TypeError, "'z'"],
-  ["d4aaz", TypeError, "'z'"],
-  ["30313", RangeError, ""],
-  ["0g", TypeError, "'g'"],
-  ["00gg", TypeError, "'g'"],
-  ["0\x01", TypeError, "'\x01'"],
-  ["ffeed", RangeError, ""],
+  ["0", TypeError, "Invalid Character (0)"],
```

Comment:
The existing error message feels better to me as it's more explicit about what's wrong. Can we change `decode` function in `_common16.ts` to be aligned with the existing behavior?

Reply:
You mean for instances where the problem is the length and not the actual character? A better error message can be added for those instances for hex, base32 and base64, but a lot of tests would need to be changed which might exceed the scope of this pull request.

Reply:
> You mean for instances where the problem is the length and not the actual character?

Yes

> A better error message can be added for those instances for hex, base32 and base64, but a lot of tests would need to be changed which might exceed the scope of this pull request.

I'd suggest we would address that only for hex for now in this PR to keep the main branch publishable state.

Reply:
The tests in unstable_hex_test.ts would also need to be updated, but I doubt that would be more than a handful.